app/assets/javascripts/discourse/components/highlight-search.js

```diff
@@ -1,12 +1,12 @@
 import Component from "@ember/component";
-import highlightText from "discourse/lib/highlight-text";
+import highlightSearch from "discourse/lib/highlight-search";
 
 export default Component.extend({
   tagName: "span",
 
   _highlightOnInsert: function() {
     const term = this.highlight;
-    highlightText($(this.element), term);
+    highlightSearch($(this.element), term);
```

Comment:
Could we try to avoid using jquery while we are at it ? I see highlight-html is using the node and not jquery object.